Source/WebCore/platform/graphics/gbm/GraphicsContextGLGBM.cpp

```diff
@@ -27,7 +27,7 @@
 #include "config.h"
 #include "GraphicsContextGLGBM.h"
 
-#if ENABLE(WEBGL) && USE(LIBGBM) && USE(ANGLE)
+#if ENABLE(WEBGL) && USE(LIBGBM) && USE(ANGLE) && USE(TEXTURE_MAPPER_DMABUF)
```

Comment:
USE(TEXTURE_MAPPER_DMABUF) is not meaningful here, this class doesn't use TextureMapper at all.

Reply:
Thanks for the comments, we are reviewing the patch to add the runtime mode for the fallback backend directly in this patch.